kahuna/public/js/components/gr-collections-panel/gr-collections-panel-node.html

```diff
@@ -19,31 +19,48 @@
         </button>
 
         <a class="node__name flex-spacer"
-           ui:sref="search.results({query: (node.data.name)})">
+           ui:sref="search.results({query: ctrl.getCollectionQuery(node.data.content.pathId)})">
             {{node.data.name}}
         </a>
 
+
         <button class="node__action clickable" type="button"
                 ng:if="ctrl.deletable"
-                ng:click="ctrl.remove()">
+                ng:show="ctrl.editing"
```

Comment:
The logic was that if it is a show hide mechanism it should be `ng:show` i.e. show hiding children. If it was logic around whether something should exist we should use `ng:if` i.e. the delete button.

It's also to get around the fact the `ng:if` creates it's own scope.